apps/vscode/README.md

```diff
@@ -5,3 +5,11 @@ It is recommended to install the [VSCode talon extension pack](https://marketpla
 ## Cursorless
 
 If you'd like to use Cursorless, follow the instructions in the [cursorless-talon repo](https://github.com/pokey/cursorless-talon).
+
+## Terminal
+
+By default the VSCode terminal is not going to recognize terminal commands from talon. In order for talon to configure terminal commands in VSCode the window has to include a word that it can use to match. This is possible using the VSCode setting to include word terminal in the title. You can do this with this setting in VSCode:
+
+```
+"window.title": "${dirty}${activeEditorShort}${separator}${rootName}${separator}${profileName}${separator}${appName}${separator}focus:[${focusedView}]",
+```
```

Comment:
````suggestion
```

This setting will cause VSCode to include a special string in the window title whenever the terminal is focused. Talon will look for this string in the window title and activate the terminal commands in response.

Note that if you have customizations in your window title that you'd like to keep, the important part is just to ensure that `focus:[${focusedView}]` appears somewhere within your custom window title.
````